File: backend/src/main/java/com/backend/ord/domain/dto/BankGroupDTO.java
package com.backend.ord.domain.dto;

import com.backend.ord.domain.dto.abstracts.DTOBase;
import lombok.*;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
@EqualsAndHashCode(callSuper = true)
public class BankGroupDTO extends DTOBase {
    private String name;
    private String color;
}

File: backend/src/main/kotlin/com/backend/ord/domain/dto/BankGroupDTO.kt
package com.backend.ord.domain.dto

import java.time.Instant
import java.util.*

data class BankGroupDTO(
    val id: UUID = UUID.randomUUID(),

    var name: String,
    var color: String,

    val createdAt: Instant = Instant.now(),
    var updatedAt: Instant = Instant.now(),
)
